src/josepy/jws.py

```diff
@@ -57,21 +57,21 @@ class Header(json_util.JSONObjectWithFields):
     :ivar str cty: Content-Type, inc. :const:`MediaType.PREFIX`.
 
     """
-    alg: jwa.JWASignature = json_util.field(
+    alg: Optional[jwa.JWASignature] = json_util.field(
         'alg', decoder=jwa.JWASignature.from_json, omitempty=True)
-    jku: bytes = json_util.field('jku', omitempty=True)
+    jku: Optional[bytes] = json_util.field('jku', omitempty=True)
     jwk: jwk_mod.JWK = json_util.field('jwk', decoder=jwk_mod.JWK.from_json, omitempty=True)
```

Comment:
Since no `default` is given here, should this type also be `Optional`?

Reply:
Totally! I fixed that.